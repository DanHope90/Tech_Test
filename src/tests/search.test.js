import React from "react";
import Render, { render } from "@testing-library/react";
import Search from "../components/search";

describe('Search', () => {
  const { asFragment } = render(<Search />);

  it('renders correctly', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
