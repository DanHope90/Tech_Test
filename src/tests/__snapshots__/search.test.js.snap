// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<DocumentFragment>
  <div
    class="search-input"
  />
  <input
    type="text"
  />
</DocumentFragment>
`;
